sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/Models/TargetBackupStorageRedundancy.cs

```diff
@@ -12,12 +12,12 @@ namespace Microsoft.Azure.Management.Sql.Models
 {
 
     /// <summary>
-    /// Defines values for DatabaseState3.
+    /// Defines values for TargetBackupStorageRedundancy.
```

Comment:
I *think* this is a breaking change as well. Although hard to tell from the diff.